### ISSUE TITLE
Un-aliased expression over 30 characters throws error.

### ISSUE THREAD

**Issue by jonheller1** [method5/method5#40]
Expressions without an alias that are more than 30 characters will throw an error in 11g (and will probably fail at 128 in 12.2):

```
select * from table(m5(q'[select sys_context('userenv', 'server_host') from dual]', 'db1,db2'));

ORA-20027: The SELECT statement was not valid, please check the syntax and that the objects exist.  The statement was tested on db1,db2.  If the objects only exist on a small number of databases you may want to run Method5 first with P_TARGETS set to one database that has the objects.  The SQL raised this error: 
ORA-01948: identifier's name length (36) exceeds maximum (30)
ORA-06512: at "SYS.DBMS_UTILITY", line 574
ORA-06512: at "METHOD5.M5_PKG", line 1573
ORA-06512: at "METHOD5.M5_PKG", line 2176
ORA-06512: at "METHOD5.METHOD4_M5_POLL_TABLE_OT", line 211
ORA-06512: at line 4

View program sources of error stack?

```

**Comment by jonheller1** [comment]
For expressions larger than 30 characters this might be a solution: https://stackoverflow.com/a/44210598/409172

**Comment by jonheller1** [comment]
Fixed in 8.6.1.
